Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -49,24 +49,115 @@ script:
         return url;
     };
 
+    var isObjectEmpty = function (obj) {
+        return Object.keys(obj).length === 0;
+    }
+
+    var mapTypeToValue = function (fields) {
+        var result = {};
+        var TYPE = 'type';
+        var VALUE = 'value';
+        if(fields && Array.isArray(fields)) {
+            fields.forEach(function (element) {
+                if (element.hasOwnProperty(TYPE) && element[TYPE]) {
```

Comment:
Because It check if value of "TYPE" property is not undefined, not empty string and not null